lib/opensusebasetest.pm

```diff
@@ -914,10 +914,10 @@ sub wait_boot {
         $self->handle_pxeboot(bootloader_time => $bootloader_time, pxemenu => 'pxe-custom-kernel', pxeselect => 'pxe-custom-kernel-selected');
     }
     # When no bounce back on power KVM, we need skip bootloader process and go ahead when 'displaymanager' matched.
-    elsif (get_var('OFW') && (check_screen('displaymanager', 5))) {
+    elsif (get_var('OFW') && (check_screen('displaymanager', 10))) {
     }
-    # SLE-16 boot on ppc64le results too quick to be captured, from grub2 to login prompt
-    elsif (get_var('OFW') && is_sle('16+') && is_ppc64le && (check_screen('linux-login', 10))) {
+    # SLE-16 on ppc64le/aarch64 sometime results too quick to be captured, from grub2 to login prompt
+    elsif (is_sle('16+') && (is_aarch64 || (is_ppc64le && get_var('OFW'))) && (check_screen('linux-login', 30))) {
```

Comment:
As we're increasing a timeout that also affects ppc64le, we need a VR for that archictecture too.

Reply:
To make understand that the previous timeout was too small that we needed to add TIMEOUT_SCALE to allow this screen to be detected, but this way increasing also all the other timeouts, it takes too long discussions: I separated the conditions for each architecture.